testsuite/features/support/commonlib.rb

```diff
@@ -194,7 +194,9 @@ def suse_host?(name)
 end
 
 def slemicro_host?(name)
-  (name.include? 'slemicro') || (name.include? 'micro')
+  node = get_target(name)
+  _os_version, os_family = get_os_version(node)
```

Comment:
```suggestion
  os_family = node.os_family
```

Or you could even use this attribute in the rest of mentions...

```suggestion
    (name.include? 'slemicro') || (name.include? 'micro') || node.os_family.include?('sle-micro') || node.os_family.include?('suse-microos')
```